src/views/GovernanceTaskList/SideNavActions/index.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import SideNavActions from './index';
+
+const renderComponent = () => {
+  const mockStore = configureMockStore();
+  const store = mockStore({});
+  return mount(
+    <MemoryRouter initialEntries={['/']} initialIndex={0}>
+      <Provider store={store}>
+        <SideNavActions />
+      </Provider>
+    </MemoryRouter>
+  );
+};
+
+describe('The TaskListSideNavActions', () => {
+  it('renders without crashing', () => {
+    const component = renderComponent();
+    expect(component.find(SideNavActions).exists()).toBe(true);
```

Comment:
There's another way I've seen to write these tests. Not a requirement to change it, but just sharing other examples...
```
expect(renderComponent()).not.toThrow()
```

Docs on `toThrow` are here: https://jestjs.io/docs/en/expect#tothrowerror